### PR TITLE
Feat/10 profile settings page

### PR DESCRIPTION
**Pull request by phoenixdahdev**
# Description

**Closes #issue_number_here**

# Changes Proposed

## What Did You Do?

# Checklist (Check all applicable boxes)

🚨 Please review the [contribution guidelines](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR are clear and explain the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages adhere to the requested style.
- [x] My code changes do not cause linting errors or failing unit tests.
- [x] I am only making changes to files I was requested to modify.

# Screenshots/Videos

Screenshot_2024-07-20_11-40-00
Screenshot_2024-07-20_11-49-21

### CODE REVIEW

File: src/app/globals.css
Comment (prudentbird) [inline src/app/globals.css:1]: Use the previous global CSS

**Review by prudentbird — CHANGES_REQUESTED** [review]
Write tests !

File: src/app/globals.css
Comment (phoenixdahdev) [reply src/app/globals.css:1]: that one has changed the bg color

Comment (prudentbird) [reply src/app/globals.css:1]: Default color isn't there

Comment (prudentbird) [reply src/app/globals.css:1]: Seen

**Review by prudentbird — CHANGES_REQUESTED** [review]
Tests please, and attach screenshots for ESLINT and tests passed

**Comment by phoenixdahdev** [comment]
> Tests please, and attach screenshots for ESLINT and tests passed

Screenshot_2024-07-20_11-59-01

**Review by prudentbird — APPROVED** [review]

**Review by Samadeen — APPROVED** [review]

**Review by Samadeen — APPROVED** [review]

**Pull request merged**